tutorials/docker-gcplogs-driver/index.md

```diff
@@ -0,0 +1,171 @@
+
+        docker run -d --name mysite --log-driver=gcplogs -p 80:80 nginx
+
+    You can specify additional options with the `--log-opt` command-line
+    argument. This command tells Docker to also log the command that the container
+    was started with.
+
+        docker run -d \
+            --name mysite \
+            --log-driver=gcplogs \
+            --log-opt gcp-log-cmd=true \
+            -p 80:80 \
+            nginx
+
+1.  Make a request to your container so that it logs something.
+
+        curl 127.0.0.1:80
+
+## Configuring the Google Cloud logging driver with Docker Compose
+
+When using Docker Compose, specify a logging driver for each service in the
+`docker-compose.yml` configuration file. For example:
+
+    version: '2'
+    services:
+      web:
+        logging:
+          driver: gcplogs
+        ...
+      database:
+        logging:
+          driver: gcplogs
+        ...
+
+## Viewing your logs
+
+Now that you are uploading logs to Stackdriver Logging, you can view them in
+the [Google Cloud Platform Console Logs
+Viewer](https://console.cloud.google.com/logs/viewer).
+
+1.  Open the [Google Cloud Platform Console Logs
+    Viewer](https://console.cloud.google.com/logs/viewer) via that link or by
+    opening **Logging** from the left menu.
+
+    ![logging menu](logging-menu.png)
+
+2.  Select the **Global** logs.
+
+    ![global logs](logging-global.jpg)
+
+3.  Select the `gcplogs-docker-driver` label to limit to just the logs from your
+    Docker containers.
+
+    ![gcplogs docker driver logs](logging-driver.jpg)
+
+4.  Enter a search filter to narrow the logs further. Enter
+    `jsonPayload.container.name:nginx-proxy` to limit to logs just from containers
+    with the name `nginx-proxy`.
+
+    ![logging container name filter](logging-container-name.jpg)
+
+## Setting the default logging driver
+
+Instead of configuring the driver for each container, you can configure Docker
+to use the Google Cloud logging driver by default.
+
+To set the Google Cloud logging driver as the [default Docker logging
+driver](https://docs.docker.com/engine/admin/logging/overview/#configure-the-default-logging-driver-for-the-docker-daemon),
+specify the `--log-driver=gcplogs` option in the `dockerd` command.
+
+    dockerd --log-driver=gcplogs
+
+Container-Optimized OS starts [Docker using
+systemd](https://docs.docker.com/engine/admin/systemd/).  To configure Docker
+to use the Google Cloud logging driver when it is started by systemd:
+
+1.  Create or edit `/etc/docker/daemon.json`.
+
+        sudo vim /etc/docker/daemon.json
+
+    Note that the `vim` command comes with Container-Optimized OS. To use other
+    editors, use the bundled [CoreOS
+    toolbox](/container-optimized-os/docs/how-to/toolbox).
+
+1.  Write the following contents:
+
+        {
+          "log-driver": "gcplogs"
+        }
+
+1.  Restart the docker service.
+
+        sudo systemctl stop docker
+        sudo systemctl start docker
```

Comment:
Done, thanks.

I wasn't sure if it would reload the config. Looks like it does.